Lib/asyncio/locks.py

```diff
@@ -78,12 +78,14 @@ def __init__(self, *, loop=None):
         self._waiters = None
         self._locked = False
         if loop is None:
-            self._loop = events.get_event_loop()
+            self._loop = events._get_running_loop()
+            if self._loop is None:
+                warnings.warn("The creation of asyncio objects outside a running "
```

Comment:
Grammar fix: should be "outside of a" rather than "outside a".

```suggestion
                warnings.warn("The creation of asyncio objects outside of a running "
```

(also applies to other warnings)